### PR TITLE
ipython: use python@3.8 and bump revision

### PR DESCRIPTION
**Pull request by drrlvn**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
Progress #47274 by updating ipython.

### CODE REVIEW

**Comment by SMillerDev** [comment]
I think the install block should also be changed to use python3.8 instead of just python3.

**Comment by drrlvn** [comment]
> I think the install block should also be changed to use python3.8 instead of just python3.

Oops, you're right. I think (hope) it's fixed now. Sorry, this is my first contribution to brew.

File: Formula/ipython.rb [hunk 1]
```diff
@@ -118,20 +119,20 @@ class Ipython < Formula
   end
 
   def install
-    xy = Language::Python.major_minor_version "python3"
+    xy = Language::Python.major_minor_version "python3.8"
```
Comment (iMichka) [inline Formula/ipython.rb:122]: xy = Language::Python.major_minor_version Formula["python@3.8"].opt_bin/"python3"

File: Formula/ipython.rb [hunk 2]
```diff
@@ -118,20 +119,20 @@ class Ipython < Formula
...
     ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python#{xy}/site-packages"
 
     # install other resources
     ipykernel = resource("ipykernel")
     (resources - [ipykernel]).each do |r|
       r.stage do
-        system "python3", *Language::Python.setup_install_args(libexec/"vendor")
+        system "python3.8", *Language::Python.setup_install_args(libexec/"vendor")
```
Comment (iMichka) [inline Formula/ipython.rb:129]: system Formula["python@3.8"].opt_bin/"python3"

File: Formula/ipython.rb [hunk 3]
```diff
@@ -118,20 +119,20 @@ class Ipython < Formula
...
       end
     end
 
     # install and link IPython
     ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python#{xy}/site-packages"
-    system "python3", *Language::Python.setup_install_args(libexec)
+    system "python3.8", *Language::Python.setup_install_args(libexec)
```
Comment (iMichka) [inline Formula/ipython.rb:135]: sytem Formula["python@3.8"].opt_bin/"python3"

File: Formula/ipython.rb [hunk 4]
```diff
@@ -140,7 +141,7 @@ def install
 
     # install IPyKernel
     ipykernel.stage do
-      system "python3", *Language::Python.setup_install_args(libexec/"vendor")
+      system "python3.8", *Language::Python.setup_install_args(libexec/"vendor")
```
Comment (iMichka) [inline Formula/ipython.rb:144]: system Formula["python@3.8"].opt_bin/"python3"

**Review by iMichka — CHANGES_REQUESTED** [review]

**Comment by drrlvn** [comment]
@iMichka Thanks! I pushed a commit with your suggested changes.

**Comment by bayandin** [comment]
Superseded by https://github.com/Homebrew/homebrew-core/pull/52781

**Pull request closed without merging**